src/Yoti.Auth/Anchors/Anchor.cs

```diff
@@ -40,7 +40,7 @@ public Anchor(AttrpubapiV1.Anchor protobufAnchor)
             _signedTimeStamp = new SignedTimestamp(protobufSignedTimestamp);
         }
 
-        private List<X509Certificate2> ConvertRawCertToX509List(RepeatedField<ByteString> rawOriginServerCerts)
+        private static List<X509Certificate2> ConvertRawCertToX509List(RepeatedField<ByteString> rawOriginServerCerts)
```

Comment:
Not sure how this will turn out in C#, but in Java static code causes problems for unit testing - it's harder to mock it

Reply:
We can overcome this using shims & MS.Fakes 👍